.github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: 💭 Proposal
-about: Suggest an idea for a specific feature you wish to propose to the community for comment
+about: Suggest an idea for a specific feature you wish to propose to the community for comment.
```

Comment:
I swear there were periods in here and we went through and took them all out.

Reply:
Ok, I'll remove them again.